holonote/annotate/plotting.py

```diff
@@ -0,0 +1,505 @@
+from __future__ import annotations
```

Comment:
I don't think 'plotting' is the right term here...should probably be something like 'display.py' instead. These are annotations that go on top of plots, not the plots themselves (or that is at least how I think most users will think about it)